app/src/androidTest/java/ch/epfl/favo/auth/LoginFlowTest.java

```diff
@@ -0,0 +1,186 @@
+//package ch.epfl.favo.auth;
```

Comment:
whats up with this file, needed?

Reply:
It's a general login test class that contains sign-in tests when offline and a general facebook login test with test credentials in normal conditions. Tests pass locally but not on Cirrus, I left it here because I hope someone will make it work in the future and will be able to increase the coverage. I might work on it in the next sprint as well.